development.env

```diff
@@ -0,0 +1,7 @@
+STARKNET_NETWORK=alpha-goerli
+STARKNET_WALLET=starkware.starknet.wallets.open_zeppelin.OpenZeppelinAccount
+STARKNET_ACCOUNT_DIR=~/.starknet_accounts_devnet
```

Comment:
If we set `STARKNET_ACCOUNT_DIR=$PWD` then we will not need to copy the account json file and this way we also avoid copying over the original account json